src/cloud-api-adaptor/test/provisioner/trustee_kbs.go

```diff
@@ -477,11 +495,11 @@ func (p *KeyBrokerService) EnableKbsCustomizedAttestationPolicy(customizedOpaFil
 func (p *KeyBrokerService) SetSampleSecretKey() error {
 	kbsClientDir := filepath.Join(TRUSTEE_REPO_PATH, "target/release")
 	privateKey := "../../kbs/config/kubernetes/base/kbs.key"
-	platform, err := getHardwarePlatform()
+	overlaysPath, err := getOverlaysPath()
 	if err != nil {
 		return err
 	}
-	keyFilePath := "../../kbs/config/kubernetes/overlays/" + platform + "/key.bin"
+	keyFilePath := "../../kbs/config/kubernetes/" + overlaysPath + "/key.bin"
```

Comment:
Is there any way to use absolute path for privateKey and keyFilePath based on the TRUSTEE_REPO_PATH?

Reply:
I started writing this code, but then noticed `const TRUSTEE_REPO_PATH = "../trustee"`, so the TRUSTEE_PATH isn't absolute. I'm happy to continue the refactor to reduce the number of relative paths we have though?

Reply:
It looks like the refactor might have broken the tests too, so I'll look into this locally

Reply:
Based on your comment, it looks like a general refactor is needed to use absolute path. How about you merge this  and handle the refactor as a separate PR ?

Reply:
Yeah, I think this commit does help reduce relative paths of relatives paths (in the old code we'd have something like `../../trustee/../../<file>`) so we just have the single relative path here now to `../trustee` from `test/e2e`, but there is room for improvement later. I think I already have 1/2 refactors in this area already, so we can steadily get it better.

Reply:
I decided to just take on the battle here, so have re-work the commit to remove the relative trustee path too. It relies on `NewKeyBrokerService` to set it up initially, but as all the references to the trustee path are in that func, or methods of `KeyBrokerService` I think we'll be okay.